### PR TITLE
AppTP: Change endpoint for blocklist to pull new format

### PR DESCRIPTION
**Pull request by laghee**
Task/Issue URL: https://app.asana.com/0/0/1204239971719179/f

### Description
Now that we're using the cumulative blocklist for AppTP, we've decided to use a new endpoint to support the new list format and any future updates. We'll maintain the old endpoint, without updates, for users on earlier versions.

### Steps to test this PR
The endpoint is up & running with the old (current) list, but in the new format (which I'll alter today to include entity scores).

To test:
Install app from `develop`, enable AppTP & let block some trackers
Update app from this branch, verify that database transitions smoothly without trying to migrate to a new version

### CODE REVIEW

**Comment by laghee** [comment]
Just realized I probably need to regenerate the test `json` here to include entity scores as well.

File: app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/trackers/AppTracker.kt [hunk 1]
```diff
@@ -105,7 +105,7 @@ data class JsonAppBlockingList(
 
 class JsonAppTracker(
     val owner: TrackerOwner,
-    val app: TrackerApp,
+    // val app: TrackerApp,
```
Comment (aitorvs) [inline app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/trackers/AppTracker.kt:108]: remove?
```suggestion
```

**Review by aitorvs — COMMENTED** [review]
Added one nit. Still need to test but we'll get to the test tomorrow

File: app-tracking-protection/vpn-store/src/test/resources/full_app_trackers_blocklist.json
Comment (aitorvs) [inline app-tracking-protection/vpn-store/src/test/resources/full_app_trackers_blocklist.json:1]: This file seems to be different than the one we have in the `/android-tds.json` endpoint. Should it not be the same?

Comment (laghee) [reply app-tracking-protection/vpn-store/src/test/resources/full_app_trackers_blocklist.json:1]: There's an open PR there to revert the entities to have scores & the version is different, but the rest should be the same. I'll ping Konrad to get that merged asap.

Comment (laghee) [reply app-tracking-protection/vpn-store/src/test/resources/full_app_trackers_blocklist.json:1]: Ah, no -- this was messed up, sorry. Fixed the list (again) -- should also match the endpoint now.

**Review by aitorvs — APPROVED** [review]
Tested app update from develop and fresh install. LGTM

**Pull request merged**